backend/src/main/java/com/team4/backend/TestingInserterRunner.java

```diff
@@ -124,6 +121,7 @@ private void insertStudents() {
                         .hasValidCv(true)
                         .appliedOffersId(new HashSet<>())
                         .exclusiveOffersId(new HashSet<>())
+                        .interviewsDate(new TreeSet<>())
```

Comment:
Why TreeSet ?

Reply:
to order date automatically it helps so i dont need to order it when i need to find the closest date